plugins/uefi-dbx/fu-uefi-dbx-device.c

```diff
@@ -212,6 +213,12 @@ fu_uefi_dbx_device_init(FuUefiDbxDevice *self)
 	fu_device_add_private_flag(FU_DEVICE(self), FU_DEVICE_PRIVATE_FLAG_HOST_FIRMWARE_CHILD);
 	if (!fu_common_is_live_media())
 		fu_device_add_flag(FU_DEVICE(self), FWUPD_DEVICE_FLAG_UPDATABLE);
+	/* dbx changes are expected to change, warn the user that BitLocker might ask for
+	recovery key after fw update */
+	if (!fu_common_check_full_disk_encryption(&error_fde)) {
```

Comment:
I think _init() is probably the wrong place for this; I think either _probe() or _setup() is more appropriate. _init() is supposed to be "initialize the object" and not "do stuff" you if you see that I mean. I guess fu_common_is_live_media() probably falls in the same boat, but it's not really failable like fu_common_check_full_disk_encryption().

Reply:
Makes sense - I put it next to the other add_flag calls thinking it would be logically grouped. 

I'll figure out how to make it work in _probe tonight